cam0:
  T_cam_imu:
  - [0.999975, 0.007038, -0.000812, -0.037983]
  - [-0.007035, 0.999969, 0.003395, 0.002513]
  - [0.000836, -0.003389, 0.999994, -0.001260]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  cam_overlap: [1]
  camera_model: pinhole
  distortion_coeffs: [0.008701, -0.018283, 0.038756, -0.025935]
  distortion_model: equidistant
  intrinsics: [469.650280, 469.551357, 375.629402, 236.808264]
  resolution: [752, 480]
  rostopic: /cam0/image_raw
cam1:
  T_cam_imu:
  - [0.9999722760516375, 0.007329193771005421, 0.0013153124248847282, 0.0790982900835488]
  - [-0.007335059837348439, 0.9999629194070246, 0.004511840884063492, 0.003549628903031918]
  - [-0.0012821954962168199, -0.00452136369336114, 0.9999889565615523, -0.01713313929463862]
  - [0.0, 0.0, 0.0, 1.0]
  T_cn_cnm1:
  - [0.9999938014675284, 0.003389608748506475, 0.0009526694354431359, 0.11007706750982221]
  - [-0.003387721466817517, 0.9999923098946835, -0.001975726387548898, -0.0005323913855882944]
  - [-0.0009593590487627364, 0.0019724867622474985, 0.9999975944602008, -0.0004467314253999674]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0]
  camera_model: pinhole
  distortion_coeffs: [0.00823121215582322, -0.015270152487108836, 0.03085334360639285,
    -0.017760720995454376]
  distortion_model: equidistant
  intrinsics: [456.3683366282091, 455.03924786357857, 375.1783411236692, 238.22971133267725]
  resolution: [752, 480]
  rostopic: /cam1/image_raw
rgb:
  T_rgb_cam0:
  - [0.999968, 0.003285, -0.007248, -0.001824]
  - [-0.003218, 0.999953, 0.009175, -0.028671]
  - [0.007277, -0.009152, 0.999932, -0.020377]
  - [0.000000, 0.000000, 0.000000, 1.000000]
  distortion_coeffs: [0.242123, -0.853689, -0.001582, 0.000793, 0.449361]
  distortion_model: radtan 
  intrinsics: [619.574765, 615.488155, 322.423351, 232.460831] 
  resolution: [640, 480] 
  rostopic: /picoflexx/cam0/depth/image_raw
depth: 
   distortion_coeffs: [0.0, 0.0, 0.0, 0.0, 0.0]
   distorion_model: radtan
   depth_correction: [-0.025884, 0.024701]
   intrinsics: [456.81292724609375, 456.81292724609375, 241.26553344726562, 178.0287322998047]
   resolution: [480, 360]
vicon:
  T_vicon_cam:
  - [0.999885, 0.014491, -0.004467, 0.053458]
  - [-0.014337, 0.999362, 0.032704, -0.006089]
  - [0.004938, -0.032636, 0.999455, -0.003491]
  - [0.000000, 0.000000, 0.000000, 1.000000]
